src/core/transpile.jl

```diff
@@ -115,24 +115,20 @@ function as_universal_gate(target::Integer,op::AbstractOperator)::Universal
 end
 
 # compress (combine) several single-target gates with a common target to a Universal gate
-function compress_to_universal(gates::Vector{<:AbstractGate})::Universal
+function compress_to_universal(gates::Vector{<: AbstractGateSymbol}, target::Int)::Gate
```

Comment:
Input of `gates::Vector{<: AbstractGateSymbol}` would fail if the elements are not (placed) Gate types.